README_CODE.md

```diff
@@ -24,19 +29,19 @@ the "models"), then renders the result via HTML templates (the "views").
 
 Rails applications run in one of three different modes: development, production
 and test.  Each has its own entirely separate database (named appropriately
-+observer_development+, +observer_production+ and +observer_test+).
+`observer_development`, `observer_production` and `observer_test`).
```

Comment:
I assume you meant backticks instead of plus signs. 
If want underscored text, we'll need inline html: `<ins>blah</ins>`

Reply:
Those are actually from sections from before that I didn't change.  The edits are welcome.